wxGUI/psmap: fix show correct error message if Ghostscript isn't installed (required for rendering output PDF file) on the OS MS Windows platform

tmszi opened the pull request:
**Describe the bug**
Wrong error message if Ghostscript gswin32c/gswin64c isn't installed (required for rendering output PDF file) on the OS MS Windows platform

**To Reproduce**
Steps to reproduce the behavior:

1. Download and install e.g. [WinGRASS-8.3.dev ](https://wingrass.fsv.cvut.cz/grass83/WinGRASS-8.3.dev-c7495e2ef-123-Setup.exe)
1. Launch wxGUI Cartographic Composer `g.gui.psmap`
2. Add Map frame and choose some raster map e.g. elevation
3. Push toolbar Generate PDF output tool
4. See error

![wxGUI_psmap_error_message_dialog_def](https://user-images.githubusercontent.com/50632337/172553519-50683c4c-62f0-43ea-a63c-a6d3de4d61e1.png)

**Expected behavior**
If Ghostscript gswin32c/gswin64c isn't installed (required for rendering output PDF file), show correct error message.

![wxGUI_psmap_error_message_dialog_expected](https://user-images.githubusercontent.com/50632337/172553677-724c4800-53f8-40e6-94a4-19f15ebb4f6b.png)

**System description:**

- Operating System: MS Windows
- GRASS GIS version: all versions

petrasovaa commented on gui/wxpython/psmap/frame.py:
```diff
@@ -421,14 +427,30 @@ def OnCmdDone(self, event):
                 ]
+                message = (
+                    "Program {} is not available."
+                    " You can donwload {} version here"
```
```suggestion
                    " You can download {} version here"
```

petrasovaa commented on gui/wxpython/psmap/frame.py:
```diff
@@ -421,14 +427,30 @@ def OnCmdDone(self, event):
                     event.userData["filename"],
                     event.userData["pdfname"],
                 ]
+                message = (
```
The message needs to be translatable here, not on line 466 below.

tmszi commented:
All right, I fixed it.

petrasovaa commented:
> All right, I fixed it.

I apologize, looking at it again, I gave you a wrong suggestion, the translation doesn't work with f-strings so you will need to use a format function as before: 
`_(xxxx {yyyy} xxx).format(...)`

tmszi commented:
> > All right, I fixed it.
> 
> I apologize, looking at it again, I gave you a wrong suggestion, the translation doesn't work with f-strings so you will need to use a format function as before: `_(xxxx {yyyy} xxx).format(...)`

Okay, I fixed it.

petrasovaa reviewed (approved):
Thanks!

tmszi commented:
@petrasovaa Maybe should be better use `HyperlinkDialog` if Ghostscript is not available (for rendering PDF), same as in PR #2421 (for rendering preview) with same simplified error message?

petrasovaa commented:
> @petrasovaa Maybe should be better use `HyperlinkDialog` if Ghostscript is not available (for rendering PDF), same as in PR #2421 (for rendering preview) with same simplified error message?

I agree, but I will leave it up to you if you want to change that.

petrasovaa commented on gui/wxpython/psmap/frame.py:
```diff
@@ -447,13 +462,20 @@ def OnCmdDone(self, event):
+                        self,
+                        title=title,
+                        message=message + str(e),
+                        hyperlink="https://www.ghostscript.com/releases/gsdnld.html.",
```
```suggestion
                        hyperlink="https://www.ghostscript.com/releases/gsdnld.html",
```

tmszi replied:
Thanks.
